### ISSUE TITLE
Installer doesn't work on R/O filesystem

### ISSUE THREAD

**Issue by syslogic** [opencart/opencart#200]
install/controller/step_2.php
install/controller/step_3.php

It just tells me "Warning: admin/config.php needs to be writable for OpenCart to be installed!"

In fact - when deploying to clouded servers configs just get pushed ...

Would be great to have any config-option to override the R/W checks (just need to install the database).

Something like step_2_ro.php & step_3_ro.php - without the checks and the file I/O.

Basically it just would need to be checked if the config files are present.

Here's a version of the installer that works in a clouded environment:
https://github.com/syslogic/opencart-quickstart/tree/master/pagoda

Here one can see it in action:
https://pagodabox.com/cafe/martin/opencart

**Comment by heldersepu** [comment]
It will be nice if the version of the pagoda installer can be updated with the V2.0 of OpenCart

**Comment by syslogic** [comment]
well, Pagoda v1 has been terminated and together with it all my containers went down -
think it's a paid service now and never received any commission from my quickstarts.

**Comment by heldersepu** [comment]
Pagoda still offers a free service(10MB writable storage) :
https://pagodabox.com/cloud-hosting-price

If the Pagoda QuickStarts work as advertised, then that is by far the easiest way to get an OpenCart store up and running. 
They already have Magento, I would think you guys want to compete with them...

About financial compensation, hit me up directly: HelderSepu(at)gmail.com 
...and we can chat about it
